spec/lib/api/reminders_spec.rb

```diff
@@ -0,0 +1,99 @@
+require 'spec_helper'
+require 'uber'
+
+describe Uber::API::Reminders do
+  let!(:client) { setup_client }
+
+  describe "get reminder" do
+    before do
+      # From: https://developer.uber.com/docs/rides/api/v1-reminders-get
+      stub_uber_request(:get, 'v1/reminders/my_reminder',
+                        {"reminder_status"=>"pending", "reminder_id"=>"48ee034e-311d-41ea-9794-02fc8dcd8696", "reminder_time"=>1473296851,
+                         "event"=>{"latitude"=>nil, "time"=>1473507651, "name"=>nil, "longitude"=>nil, "location"=>nil}, "product_id"=>{}}
+      )
+    end
+
+    it "should return details about reminder" do
+      reminder = client.reminder('my_reminder')
```

Comment:
this `'my_reminder` string is the reminder id, but the stubbed response on [line 11](https://github.com/sishen/uber-ruby/pull/23/files#diff-8120bcc66d9fb49e80f2e0649373e549R11)  has a different reminder_id in the response. mind making them the same, just for clarity? :)